package.json

```diff
@@ -22,7 +22,9 @@
     "svg:win": "if not exist dist md dist && yarn svg",
     "svg": "svg-sprite-generate -d src/assets/ -o dist/sprite.svg",
     "pull_tools": "git submodule update --init --recursive",
-    "checkout_tools": "git submodule foreach git pull origin master"
+    "checkout_tools": "git submodule foreach git pull origin master",
+    "test:cypress": "cypress open",
```

Comment:
When I've run `yarn test:cypress`, there are many unnecessary example files that have been created.

Reply:
removed cypress.json from gitignore, so it will work now